File: MiniDOMBase/DMD/Sistema/Modules/CModuleSettings.vb
Imports System.Net.Mail 'importo il Namespace
Imports System.Net.Sockets
Imports minidom
Imports minidom.Databases
Imports minidom.Anagrafica

Partial Public Class Sistema

    <Serializable> _
    Public Class CModuleSettings
        Inherits CSettings

        Public Sub New()
        End Sub

        Public Sub New(ByVal owner As CModule)
            MyBase.New(owner)
        End Sub

        Public Overloads Sub Update()
            Me.Initialize(Me.Owner)
        End Sub

    End Class

End Class

File: MiniDOMBase/DMD/Sistema/Modules/Settings/CModuleSettings.cs
using System;

namespace minidom
{
    public partial class Sistema
    {
        [Serializable]
        public class CModuleSettings : CSettings
        {
            public CModuleSettings()
            {
            }

            public CModuleSettings(CModule owner) : base(owner)
            {
            }

            public void Update()
            {
                Initialize(Owner);
            }
        }
    }
}